portal-web/test/functional/com/liferay/portalweb/macros/layout/sitepage/Page.macro

```diff
@@ -218,6 +218,32 @@ definition {
 		Page.uploadFileForLogo(logoFile = "${logoFile}");
 	}
 
+	macro changeSinglePageTheme {
+		Click(locator1 = "ControlMenu#EDIT");
+		Pause(locator1 = "5000");
```

Comment:
We should avoid implicit pauses.

Reply:
We have a macro for this already, ContentPages.clickPencil(); It has a implicit pause added...